src/Tribe/Views/V2/View.php

```diff
@@ -819,18 +806,12 @@ public function prev_url( $canonical = false, array $passthru_vars = [] ) {
 		}
 
 		// Make sure the view slug is always set to correctly match rewrites.
-		$query_args['eventDisplay'] = $this->slug;
+		$url = add_query_arg( [ 'eventDisplay' => $this->slug ], $url );
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 96cd34e3251390fb3a5be9af2086fbea</sub>